tools/telemetry-generator/package.json

```diff
@@ -35,21 +35,22 @@
 		"applicationinsights": "^2.4.1"
 	},
 	"devDependencies": {
-		"@fluid-internal/mocha-test-setup": "^2.3.1",
-		"@fluidframework/build-common": "^2.0.1",
-		"@fluidframework/eslint-config-fluid": "^5.2.0",
+		"@fluid-internal/mocha-test-setup": "^2.11.0",
+		"@fluidframework/build-common": "^2.0.3",
+		"@fluidframework/eslint-config-fluid": "^5.6.0",
 		"@types/mocha": "^10.0.9",
-		"@types/node": "^14.18.0",
+		"@types/node": "^18.19.59",
 		"@types/sinon": "^17.0.3",
-		"concurrently": "^8.2.1",
+		"concurrently": "^9.1.0",
 		"eslint": "~8.55.0",
 		"mocha": "^10.7.3",
 		"mocha-multi-reporters": "^1.5.1",
 		"prettier": "~3.0.3",
-		"rimraf": "^2.6.2",
+		"rimraf": "^5.0.10",
 		"sinon": "^19.0.2",
-		"typescript": "~4.5.5"
+		"typescript": "~5.4.5"
 	},
+	"packageManager": "pnpm@8.15.8+sha512.d1a029e1a447ad90bc96cd58b0fad486d2993d531856396f7babf2d83eb1823bb83c5a3d0fc18f675b2d10321d49eb161fece36fe8134aa5823ecd215feed392",
```

Comment:
That is awfully specific. Can you comment that somewhere?

Reply:
That comment.
I am surprised `pnpm@8.15.8` isn't enough to be version specific, but okay.
Do you know when packageManager versus engines matter or interact? We just list pnpm 8 under engines.

Reply:
I don't. Maybe @tylerbutler can comment when he's back. Or when he sees this comment while OOF and can't resist the temptation to reply like I've seen him do in the past😝.

Reply:
Just hopefully not while driving :)